src/compiler/mir/ops/traverser.rs

```diff
@@ -156,7 +156,26 @@ impl<'a, L, V, T: FunctionBuilder<L, V>> FunctionTraverser<'a, L, V, T> {
                 let cond = self.operand(cond);
                 self.xfmr.term_cond_goto(cond, *tru, *fls)
             }
-            TerminatorKind::CallFn { .. } => todo!(),
+            TerminatorKind::CallFn {
+                func,
+                args,
+                reentry,
+            } => {
+                // Get the target function
+                let target = match func {
+                    Operand::Constant(_) => todo!(),
+                    Operand::LValue(l) => self.lvalue(l),
+                };
+
+                // evaluate arguments?
+                let args: Vec<_> = args.iter().map(|a| self.operand(a)).collect();
```

Comment:
This is creating vectors twice: once here and once in the FuncBuilder.  Why not pass the iterator to the FuncBuilder and then construct a vector there (and do it once).

Reply:
Passing the iterator will not work because the closure borrows `mut self` and `self.xfmr` borrows `self`, so passing the iterator to `xfmr.term_call_fn` will violate borrow rules.